tools/istio-iptables/pkg/cmd/root.go

```diff
@@ -193,17 +205,25 @@ func constructConfig() *config.Config {
 }
 
 // getLocalIP returns the local IP address
-func getLocalIP() (net.IP, error) {
+func getLocalIPs() ([]net.IP, error) {
```

Comment:
We are going from 1 IP to a lot of IPs? Is that expected?

On my machine, for example, this return 20 IP addresses.

Reply:
Hi @howardjohn, yes.  However, there are 2 purpose for this changes: 
1. Validate the network reachability (as you comments above, it just validates only 1 interface now )
2. Make sure that `EnableInboundIPv6` can be set correctly (the iteration will be break and set `EnableInboundIPv6` as true once any one IPv6 interface detected)

Reply:
If that is it then it would be better to return `net.IP, bool`; otherwise we may end up using all the other IPs which we shouldn't

Reply:
Hi @howardjohn, sure, it makes sense, I changed it back and return one more result to indicate for supporting IPv6 or not. Thanks for your comment!